ts/src/constants.ts

```diff
@@ -7,6 +7,8 @@ export const SECONDS_PER_DAY: number = 24 * SECONDS_PER_HOUR
 
 export const maxSettableEventLevelAttributionsPerSource: number = 20
 
+export const maxSettableAggregatableReportsPerSource: number = 100
```

Comment:
Where is 100 coming from? I don't see that as a constant in the specification. Or should this be a vendor-specific value?

Reply:
Moved to vendor-specific values.